pubspec.yaml

```diff
@@ -9,23 +9,15 @@ dependencies:
   flutter:
     sdk: flutter
   equatable: ^2.0.3
+  dartlin: ^0.6.1
```

Comment:
If this should serve as an example to the flame community later, maybe we shouldn't use too much magic syntax? 😄
What do we need dartlin for now?

Reply:
I wanted to use `let`, I thought it wasn't super magical, but actually to make it simpler. since it's an app (not a library) I thought adding deps was ok